### PR TITLE
item-approved-member email

### PR DESCRIPTION
**Pull request by Baggerone**
image

image

### CODE REVIEW

File: application/templates/mail/item_approved_member.plush.html [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
-
 	<p>
 		Dear <%= personFirstName %>,
 	</p>
 
 	<p>
-		Your <%= item.Name %> has been approved for coverage and added to your <%= policyType %> policy.
+		<%= item.Name %> has been approved for coverage and added to your <%= policyType %> policy.
+		<br><br>
+		&mdash;<%= supportFirstName %>
```
Comment (Schparky) [inline application/templates/mail/item_approved_member.plush.html:16]: `<br>` is frowned upon these days. I've been using a separate `p` tag for the signature line

File: application/templates/mail/item_auto_approved_steward.html [hunk 2]
```diff
@@ -11,12 +11,8 @@
...
 		coverageStartDate: coverageStartDate,
 		accountablePerson: accountablePerson,
-		householdID: policy.HouseholdID}
-	) %>
-
-	<%= partial("button", {
-		url: itemURL,
-		label: "Open in " + appName}
+		householdID: policy.HouseholdID,
+		itemURL: itemURL}
```
Comment (Schparky) [inline application/templates/mail/item_auto_approved_steward.html:15]:
```suggestion
		url: itemURL}
```

**Review by Schparky — APPROVED** [review]
Approved, assuming you change the `itemUrl` to `url` in the button instance I marked.

File: application/templates/mail/item_auto_approved_steward.html (hunk 2 above)
Comment (Baggerone) [reply application/templates/mail/item_auto_approved_steward.html:15]: I think you're thinking of the button params.  It looks like the item card expects an itemURL param.

Comment (Schparky) [reply application/templates/mail/item_auto_approved_steward.html:15]: Oh, sorry. The github diff threw me off. I was looking at the removed (red) code.

**Pull request merged**